modules/repository-s3/src/yamlRestTest/java/org/elasticsearch/repositories/s3/AbstractRepositoryS3ClientYamlTestSuiteIT.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.repositories.s3;
+
+import com.carrotsearch.randomizedtesting.annotations.Name;
+import com.carrotsearch.randomizedtesting.annotations.ParametersFactory;
+
+import org.elasticsearch.core.Booleans;
+import org.elasticsearch.test.rest.yaml.ClientYamlTestCandidate;
+import org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase;
+
+public abstract class AbstractRepositoryS3ClientYamlTestSuiteIT extends ESClientYamlSuiteTestCase {
+    static final boolean USE_FIXTURE = Booleans.parseBoolean(System.getProperty("tests.use.fixture", "true"));
```

Comment:
I actually don't think we need this. We _never_ run any tests in `yamlRestTest` without a fixture. This should only be applicable to `S3RepositoryThirdPartyTests` which are designed to run against a real endpoint.

Reply:
I kept it there to ensure all test behave like they did before. that is, if use fixture in the build script is declared, those tests are never executed.

Reply:
Yeah, but I don't think that makes sense for these tests. These tests never talk to a real external service. That's what the third party tests are for. I think this should exist there and not here.

Reply:
I think there's something fundamentally wrong in how these tests are designed in the first place already before this PR. 
With this change they never run against third party really. only against the minio fixture. Actually I think the yamlRestTest task should run against thirdparty. I create a separate ticket to address this as this has been all wrong before this